### PR TITLE
Add code-of-conduct.md

### PR DESCRIPTION
**Pull request by spiffxp**
Refer to kubernetes/community as authoritative source for code of conduct

ref: kubernetes/community#1527

### CODE REVIEW

File: code-of-conduct.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Kubernetes Community Code of Conduct
+
+Please refer to our [Kubernetes Community Code of Conduct](https://git.k8s.io/community/code-of-conduct.md)
```
Comment (ixdy) [inline code-of-conduct.md]: minor nit - add trailing period? :)

Comment (cblecker) [reply code-of-conduct.md]: This is a template so it is the exact same in all k/* repos (ref: https://github.com/kubernetes/community/issues/1527)

**Comment by ixdy** [comment]
This is going to be like the `All rights reserved` typo, isn't it? :)

/lgtm

**Pull request merged**